src/scripts/main.js

```diff
@@ -1,3 +1,39 @@
 'use strict';
 
 // write code here
+
+const nodeListLi = document.querySelectorAll('li');
+const arrLi = Array.from(nodeListLi);
+
+function sortList(list) {
+  const sortedItems = list.sort((a, b) => {
```

Comment:
The `sort` method is used directly on the `list` parameter, which sorts the array in place. If the original order of `arrLi` needs to be preserved elsewhere in the application, consider making a copy of the list before sorting.